exercises/practice/armstrong-numbers/armstrong_numbers.rb

```diff
@@ -1,7 +1,5 @@
-=begin
-Write your code for the 'Armstrong Numbers' exercise in this file. Make the tests in
-`armstrong_numbers_test.rb` pass.
-
-To get started with TDD, see the `README.md` file in your
-`ruby/armstrong-numbers` directory.
-=end
+module ArmstrongNumbers
```

Comment:
Are we removing the comment in the stubs now?  Is this no longer considered helpful?

Reply:
All these comments should be removed or amended now. They don't make sense in the context of the online editor. The website should be informing someone what to do - not a comment in a file they see.